help center: Clean up "Deactivate or reactivate a user" help page, with minor tweaks to related pages. (#22846)
zulip/zulip: templates/zerver/help/deactivate-or-reactivate-a-user.md

# Deactivate or reactivate a user

## Deactivating a user

{!admin-only.md!}

When you deactivate a user:

* The user will be immediately logged out of all Zulip sessions, including
  desktop, web and mobile apps.

* The user's credentials for logging in will no longer work, including password
  login and [any other login options](/help/configure-authentication-methods)
  enabled in your organization .

* The user's [bots](/help/bots-and-integrations) will be deactivated.

* [Email invitations and invite links](/help/invite-new-users) created by the
  user will be disabled.

* Even if your organization [allows users to join without an
  invitation](/help/restrict-account-creation#set-whether-invitations-are-required-to-join),
  this user will not be able to rejoin with the same email account.

!!! warn ""
    You must go through the deactivation process below to fully remove a user's
    access to your Zulip organization. Changing a user's password or removing
    their single sign-on account will not log them out of their open Zulip
    sessions, or disable their API keys.

### Deactivate a user

{start_tabs}

{tab|via-user-profile}

{!manage-this-user.md!}

> 1. Click the **Deactivate user** button at the bottom.

Reviewer: Maybe we can tweak this a bit and say "... at the bottom of the widget."?
```suggestion
1. Click **Deactivate user** at the bottom of the widget.

```

Author: Trying:

> Click **Deactivate user** at the bottom of the **Manage user** menu.

1. Configure **Notify this user by email?** as desired.

> 1. If notifying the user, you can choose to enter a custom comment that will be
>    included in the notification email.

Reviewer: I think we should consider combining this step with the previous step. Maybe we should start with "Select" or "Check the box" instead of "Configure", and say something like:

1. _(Optionally)_  Check the box **Notify this user by email?**, and you can also enter a custom comment to include it in the notification email.

```suggestion
1. _(Optionally)_  Check the box **Notify this user by email?**, and
   you can also enter a custom comment to include it in the notification email.
```

Author: Trying:

> 1. *(optional)* Select **Notify this user by email?** if desired, and enter a
   custom comment to include in the notification email.

1. Click the **Deactivate** button to the right of the user account that you
   want to deactivate.

1. Configure **Notify this user by email?** as desired.

> 1. If notifying the user, you can choose to enter a custom comment that will be
>    included in the notification email.

Reviewer: Same as the previous comment.

```suggestion
1. _(Optionally)_  Check the box **Notify this user by email?**, and
   you can also enter a custom comment to include it in the notification email.
```
